ksqldb-api-client/src/test/java/io/confluent/ksql/api/client/integration/ClientIntegrationTest.java

```diff
@@ -958,29 +996,24 @@ public void shouldListTopics() throws Exception {
   }
 
   @Test
-  public void shouldListQueries() {
+  public void shouldListQueries() throws ExecutionException, InterruptedException {
```

Comment:
This test was flake-prone, since it assumes that it could serialize itself after other tests (i.e., wait until all other tests' queries have stopped) and that other tests wouldn't start new queries during its execution.

There doesn't seem to be a reason for this, so I changed the test just to verify the thing it wanted to verify, regardless of what other queries are running.